lib/transport_web/views/resource_view.ex

```diff
@@ -6,10 +6,25 @@ defmodule TransportWeb.ResourceView do
   def format_related_objects(nil), do: ""
   def format_related_objects(related_objects) do
     related_objects
-    |> Enum.map(fn %{"id" => id, "name" => name} -> "#{name} (#{id})" end)
-    |> Enum.join(", ")
+    |> Enum.map(fn %{"id" => id, "name" => name} -> content_tag(:li, "#{name} (#{id})") end)
   end
 
   def issue_type([]), do: nil
   def issue_type([h|_]), do: h["issue_type"]
+
+  def template(issues) do
+    %{
+      "DuplicateStops" => "_duplicate_stops_issue.html",
+      "ExtraFile" => "_extra_file_issue.html",
+      "MissingFile" => "_missing_file_issue.html",
+      "NullDuration" => "_speed_issue.html",
+      "ExcessiveSpeed" => "_speed_issue.html",
+      "NegativeTravelTime" => "_speed_issue.html",
+      "Slow" => "_speed_issue.html",
+      "UnusedStop" => "_unused_stop_issue.html",
+      "InvalidCoordinates" => "_coordinates_issue.html",
+      "MissingCoordinates" => "_coordinates_issue.html",
+    }
+    |> Map.get(issue_type(issues.entries), "_generic_issue.html")
```

Comment:
The pipe is not needed here

Reply:
I agree, but isn’t it more readable this way ?

Reply:
Don't really have any opinion on the best readabilty between this one and 
```
Map.get(
       %{
      "DuplicateStops" => "_duplicate_stops_issue.html",
      "ExtraFile" => "_extra_file_issue.html",
      "MissingFile" => "_missing_file_issue.html",
      "NullDuration" => "_speed_issue.html",
      "ExcessiveSpeed" => "_speed_issue.html",
      "NegativeTravelTime" => "_speed_issue.html",
      "Slow" => "_speed_issue.html",
      "UnusedStop" => "_unused_stop_issue.html",
      "InvalidCoordinates" => "_coordinates_issue.html",
      "MissingCoordinates" => "_coordinates_issue.html",
    },
    issue_type(issues.entries),
    "_generic_issue.html"
)
```
Do as you wish.